wherehows-web/tests/acceptance/tracking-test.ts

```diff
@@ -0,0 +1,26 @@
+import { module, test } from 'qunit';
+import { visit } from '@ember/test-helpers';
+import { setupApplicationTest } from 'ember-qunit';
+import appLogin from 'wherehows-web/tests/helpers/login/test-login';
+
+module('Acceptance | tracking', function(hooks) {
+  setupApplicationTest(hooks);
+
+  test('Search tracks correctly', async function(assert) {
+    const searchTerm = 'somerandomsearch';
+    const paq = window && window._paq ? window._paq : [];
+    const NAME = 0;
+    const KEYWORD = 1;
+    const SEARCH_EVENT_NAME = 'trackSiteSearch';
+
+    // mocking paq qeue if not available
+    window._paq = paq;
```

Comment:
Can we move this global to a before/Each and clean up after so we don't pollute / clobber the global namespace for other tests or test modules

Reply:
Yep! Thanks